src/pudl_archiver/archivers/eiawater.py

```diff
@@ -0,0 +1,43 @@
+"""Download EIA Thermal Cooling Water data."""
+import logging
+import re
+import typing
+from pathlib import Path
+
+from pudl_archiver.archivers.classes import (
+    AbstractDatasetArchiver,
+    ArchiveAwaitable,
+    ResourceInfo,
+)
+
+logger = logging.getLogger(f"catalystcoop.{__name__}")
+BASE_URL = "https://www.eia.gov/electricity/data/water"
+
+
+class Eia923Archiver(AbstractDatasetArchiver):
```

Comment:
I think you might've forgot to change the class name

Reply:
Indeed, typo fixed!